### DISCUSSION TITLE
SMTP Settings - reset

### DISCUSSION THREAD

**Discussion by davidklemme** [zitadel/zitadel#4511] [Q&A] [answered]
Hi, 

I set up custom SMTP settings in order to use mu own Mailservice. Thats not working (custom domain stuff I presume).

Apart from how to debug best - how can I reset this to default settings?
It would be nice to have the option to use default again/as fallback, as I assume I'll have to create a new instance atm, or?

Best, 
David

**Comment by livio-a** [comment, marked as answer]
Hi @davidklemme 

There's currently no reset functionality at the moment. But we can set the default settings for you.
Please send us the ID and domain of your instance to support@zitadel.com
You find these in the instance details in the Console:
![image](https://user-images.githubusercontent.com/9405495/194810491-e74d2e31-f861-47b6-bd59-47b3f677059a.png)

Best
Livio
